tests/test_phi_finder.py

```diff
@@ -34,7 +34,7 @@ def test_phi_finder():
     xlogin = xnat4tests.connect()
 
     cmd_id = install_cs_command(
-        "ghcr.io/australian-imaging-service/quality-control.phi-finder:2025.7.2",
+        "ghcr.io/australian-imaging-service/quality-control.phi-finder:2025.8.1",
```

Comment:
Maybe here we could grab the yaml file and read the version number from that to use here